src/frontend/apps/standalone_site/src/components/Layout/MainLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import { theme } from 'lib-common';
+import React from 'react';
+import styled from 'styled-components';
+
+const colorMenu = normalizeColor('blue-active', theme);
+
+const MainLayoutBox = styled(Box)`
+  color: ${colorMenu};
+  min-height: 100vh;
+`;
```

Comment:
You can use the props `height` (`height={{min: '100vh'}}`) for the min-height.
And for the color, I wonder if we should not configure the brand color in the theme since all headers, texts etc default color is the brand one.

Reply:
What do you mean? You want to replace the "blue-active" color with the "brand" color?
Up to you after, I have just followed the design.